plugin/parser/differ/pg/test-data/test_differ_data.yaml

```diff
@@ -1,51 +1,72 @@
-- oldSchema: CREATE TABLE projects ();
+- oldSchema: |
+    CREATE TABLE public.projects ();
   newSchema: |
-    CREATE TABLE users (
+    CREATE TABLE public.users (
```

Comment:
For GitOps SDL, we should be able to handle the default schema where no schema means public schema.

I'm thinking of group things into schemas, and do a differ for each pg schema.

Reply:
But, okay for now, we can fix it (support GitOps) later.